docs/src/api/class-apirequest.md

```diff
@@ -61,7 +61,7 @@ Methods like [`method: APIRequestContext.get`] take the base URL into considerat
     - `sameSite` <[SameSiteAttribute]<"Strict"|"Lax"|"None">>
   - `origins` <[Array]<[Object]>>
     - `origin` <[string]>
-    - `localStorage` <[Array]<[Object]>>
+    - `localStorage` ?<[Array]<[Object]>>
```

Comment:
I am surprised that `origins` and `cookies` are not optional either. Maybe I was wrong suggesting this change. Perhaps we should do some archeology, or forget about it.

Reply:
Let's forget about it, we can still do this change in the future,.